prow/scripts/lib/kyma.sh

```diff
@@ -63,17 +63,21 @@ function kyma::undeploy_kyma() {
 #
 # Arguments:
 #   t - GitHub token
+#   v - searched version (optional)
```

Comment:
Looking at the code I see it accepts the regular expression for the given `tag_name` can you state some examples on which formats are accepted by this option?